kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -371,7 +475,7 @@ sealed abstract class Stream[V, -S]:
       * @return
       *   The final accumulated value
       */
-    def runFold[A, S2](acc: A)(f: (A, V) => A < S2)(using tag: Tag[Emit[Chunk[V]]], frame: Frame): A < (S & S2) =
+    def runFoldKyo[A, S2](acc: A)(f: (A, V) => A < S2)(using tag: Tag[Emit[Chunk[V]]], frame: Frame): A < (S & S2) =
```

Comment:
I wonder if clause interleaving or a `RunFoldOps` class could avoid a different name here

Reply:
I tried both and couldn't make it work. Something about having two parameters makes inference stop working as expected.